sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/reflect/DoFnSignatures.java

```diff
@@ -2511,6 +2512,10 @@ public static boolean usesBagState(DoFn<?, ?> doFn) {
     return usesGivenStateClass(doFn, BagState.class) || requiresTimeSortedInput(doFn);
   }
 
+  public static boolean usesMultimapState(DoFn<?, ?> doFn) {
+    return usesGivenStateClass(doFn, MultimapState.class) || requiresTimeSortedInput(doFn);
```

Comment:
Does requiresTimeSortedInput leverage MultimapState for its support?

Reply:
I think I misunderstood, removed it.